playwright/_impl/_transport.py

```diff
@@ -44,7 +45,7 @@ def _get_stderr_fileno() -> Optional[int]:
 class Transport(ABC):
     def __init__(self, loop: asyncio.AbstractEventLoop) -> None:
         self._loop = loop
-        self.on_message = lambda _: None
+        self.on_message: Callable[[ParsedMessagePayload], None] = lambda _: None
```

Comment:
please also change the return type in `deserialize_message` to `ParsedMessagePayload`.

Reply:
Done